docs/GDI-SOP_charter.md

```diff
@@ -0,0 +1,163 @@
+
+Through this Charter, the GDI underscores its commitment to fostering a unified yet adaptable operational environment, ensuring that all nodes, regardless of their current capabilities, can contribute to and benefit from this pan-European initiative.
+
+## 5. Purpose of this Charter
+The purpose of this document is to **define the scope and management of SOPs within the GDI network**. Specifically, it outlines how GDI SOPs will be managed to ensure consistency, quality, and relevance. This Charter serves as a foundational document within the GDI SOP quality management system, providing the necessary guidance and structure for the development, approval, and implementation of SOPs across the network.
+
+## 6. Scope of GDI SOPs
+The GDI is not a monolithic entity but a network of existing and emerging infrastructures. While SOPs can encompass various aspects of a team, node, or even project, this repository is not intended to cover every process within all institutions under the GDI umbrella. Instead, it contains **SOPs that are relevant to GDI-specific use cases** and are implemented by the nodes forming the GDI network.
+
+For example, if a GDI node already functions as an archive with submitted datasets, the curation of these datasets, when outside the GDI scope, is not expected to follow a hypothetical "Dataset Curation SOP" from GDI.
+
+Therefore, it is the **responsibility of the GDI nodes to implement these SOPs when the use case pertains to GDI infrastructure**, governance, or data.
+
+The scope of these SOPs is focused on roles, topics, and resources that are essential for establishing or operating node services, ensuring high-quality data management, and maintaining robust technical infrastructure. However, certain items are deliberately excluded from the current scope to maintain focus.
+
+### In-Scope
+#### Roles
+- **Helpdesk Staff and Operations Bioinformaticians**: Responsible for responding to and resolving user queries and issues within the node.
+- **Data Stewards, Data Curators, and Bioinformaticians**: Focused on the preparation, harmonization, and curation of data and metadata to ensure high-quality data management.
+- **Software Developers, Software Engineers, and DevOps Engineers**: Tasked with designing, developing, implementing, deploying, and monitoring software and services deployed at a node.
+
+#### Topics
+- **Data Protection & Security**: Includes procedures related to security incident response, risk assessment, and compliance with data protection regulations (e.g., DPA, DAA, DPIA).
```

Comment:
I guess that you should add these abbreviations in the glossary, especially DAA and DPIA.